container/base/bin/entrypoint.sh

```diff
@@ -65,13 +64,15 @@ if [ -f /usr/local/bin/npm-setup.sh ]; then
 fi
 
 # kludge: opinionated defaults
-#if [ ! -e ${HOME}/.local/share/code-server/User/settings.json ]; then
-#mkdir -p ${HOME}/.local/share/code-server/User
-#echo "{
-#    "workbench.colorTheme": "Abyss",
-#    "terminal.integrated.defaultProfile.linux": "bash",
-#    "telemetry.enableTelemetry": false
-#}" > ${HOME}/.local/share/code-server/User/settings.json
-#fi
+# the below is a requirement for UBI based images (and wont break antything Debian based)
+if [ ! -e ${HOME}/.local/share/code-server/User/settings.json ]; then
```

Comment:
The code below was causing some issues when I was testing... I'll test again, but those were just my opinionated settings... they can be removed :)

Reply:
The only thing that really needed to be added to make it work on OCP was the `"terminal.integrated.shell.linux": "/bin/bash",`. OCP with its random UID (even with the NSS wrapper and related ENV vars) wasn't being respected by `code-server` and was still seeing that it had its shell set to `/sbin/nologin`. This config (which apparently changes soon in Code Server to a "profile" approach) gets around that. So its possible the NSS wrapper may not even be required and can be cleaned up a little bit. I'll test that later, but for now this is a known working config (yay).

Reply:
Sounds good.